data/json/furniture_and_terrain/appliances.json

```diff
@@ -18,7 +18,7 @@
     "item": "fridge",
     "requirements": {
       "install": { "skills": [ [ "mechanics", 3 ] ], "time": "60 m", "using": [ [ "vehicle_wrench_2", 1 ] ] },
```

Comment:
I was considering removing the `install` entry since I assume it would be used to install the fridge to a vehicle (which last time I've checked is not possible). Should I do so?